code/datums/antag.dm

```diff
@@ -0,0 +1,66 @@
+/datum/antag //This file doubles as the guide to creating a new antag
```

Comment:
Ignore this entire file, I'm not even sure how this old thing got in here, you want code/datums/mode/role.dm